pages/episodes/[slug].js

```diff
@@ -22,7 +22,7 @@ export default function EpisodeDetailsPage() {
     // this logic will change when using remote API
     const [filteredEpisode] = serie
       .slice()
```

Comment:
The slice is unnecessary, since filter already returns a new array

Reply:
changed it